src/java/org/apache/cassandra/cql3/statements/schema/TableAttributes.java

```diff
@@ -185,6 +208,26 @@ private Map<String, String> getMap(Option option)
         return value;
     }
 
+    private boolean hasStringValue(Option option)
+    {
+        Object val = properties.get(option.toString());
+        if (val != null && val instanceof String)
```

Comment:
@Maxwell-Guo why do you use if / else? the result of `val != null && val instanceof String` is a boolean as such, `hasMapValue` method same case